spec/services/declarations/create_spec.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Declarations::Create, type: :model do
```

Comment:
I think we're missing a few test scenarios here:

- Output fee statements
- Superseded by
- Eligible/ineligible states